### ISSUE TITLE
unknown flag: --fargate using latest eksctl10.0.2

### ISSUE THREAD

**Issue by jguionne** [weaveworks/eksctl#1630]
**What happened?**
$ eksctl create cluster --name is2 --region us-west-2 --fargate
Error: unknown flag: --fargate

$ eksctl version
[ℹ]  version.Info{BuiltAt:"", GitCommit:"", GitTag:"0.10.2"}

**What you expected to happen?**
--fargate flag should be supported according to https://aws.amazon.com/blogs/aws/amazon-eks-on-aws-fargate-now-generally-available/

**How to reproduce it?**
See above

**Anything else we need to know?**
using macOs, updgraded eksctl with homebrew but also using curl!

**Versions**
Please paste in the output of these commands:
```
$ eksctl version
[ℹ]  version.Info{BuiltAt:"", GitCommit:"", GitTag:"0.10.2"}
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.3", GitCommit:"b3cbbae08ec52a7fc73d334838e18d17e8512749", GitTreeState:"clean", BuildDate:"2019-11-14T04:24:34Z", GoVersion:"go1.12.13", Compiler:"gc", Platform:"darwin/amd64"}
```

**Logs**.
 $ eksctl create cluster --name is2 --region us-west-2 --fargate -v 5
Error: unknown flag: --fargate

**Comment by blackdog0403** [comment]
Please check release note below. Eksctl 0.10.2 does not support fargate flag.

**Comment by cPu1** [comment]
Fargate is supported on [eksctl 0.11](https://github.com/weaveworks/eksctl/releases/tag/0.11.0-rc.0)(currently a RC but there'll be a more "stable release" today).
Fargate wasn't even released when eksctl 0.10.2 was out.

**Comment by marccarre** [comment]
Latest stable release, with support for Fargate, is available here: [`0.11.1`](https://github.com/weaveworks/eksctl/releases/tag/0.11.1).
Installation steps are documented [here](https://eksctl.io/introduction/installation/).

Please don't hesitate to let us know how your experience is once you're on the appropriate version!
Feedback is always welcome! 🙇 ✨ 

(Meanwhile, I'll close this issue as I believe this should be fine for now? But please feel free to re-open it if you still experience installation and/or version issues.)
